modules/01_3Scale_Architecture/Team1_3scale_OCP4_OCS_Install_Notes.adoc

```diff
@@ -0,0 +1,155 @@
+# NOTE
+
+These steps are an alternative install using OCS, NOT RWO Storage.
+
+# Early Notes on Running 3scale 2.10 on OCP 4.8
+### Background
+The 3scale course has been delivered so far on OCP 3.x even though 3scale has been supported on OCP 4 since 3scale 2.5.
+
+Primarily, this is due to following issues:
+
+- The 3scale template is not supported to deploy AMP on OCP. Only the 3scale operator is supported. Hence, the existing 3scale Install Ansible Roles cannot be used for deploying to OCP 4.
+- On OCP 3.x, Gluster FS is available for providing RWX storage necessary for 3scale. On OCP 4.x, no RWx storage is available on OpenTLC or RHPDS.
+- 3scale Operator does not allow customization of template used, so the resources and limits cannot be changed. This interferes with the quotas in the OpenTLC and RHPDS OCP workshop instances.
+
+# Installation Overview
+1. Request RHPDS + create project
+2. In web console, update machinesets to change 3rd set from 0 to 1 replica
+3. Install 3scale operator
+4. Install OpenShift Container Storage operator
+5. Create ClusterStorage resource using all 3 machines
+6. Create APIManager resource
+
+'''
+
+### 1. Request RHPDS + Create Project
+1. Order `OpenShift Workshop 4.8` on the [RHPDS catalog](https://rhpds.redhat.com/catalog/explorer).
+2. Confirm the warning, and leave all other default settings.
+3. Select `Training` as the Purpose.
+4. Submit - this can take about 30 minutes to finish, and you will get an email. 
+5. If the provision fails, simply repeat the previous steps.
+6. Once provisioned, `ssh` to the bastion using the SSH password provided in the email.
```

Comment:
@satyaj Team two followed my instructions of using the Web Console, Team 1 did the same thing but with a CLI. It's probably beneficial to have both, but we may need to find a way to combine them.